app/serializers/user_post_topic_bookmark_base_serializer.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_relative 'post_item_excerpt'
+
+class UserPostTopicBookmarkBaseSerializer < UserBookmarkBaseSerializer
+  include TopicTagsMixin
+
+  attributes :topic_id,
+             :linked_post_number,
+             :title,
+             :fancy_title,
```

Comment:
```suggestion
```

These have already been included in the parent serializer. Instead of overriding the methods of the parent class, what do you think about using composition over inheritance instead? I find it much clearer as compared to some methods being overrided while some are not.

Reply:
Sorry not following exactly what you are suggesting to do...maybe see my other comment about the serializers with the chat example and this might be clearer. Also thanks for pointing out the attribute duplication, will remove.

Reply:
`UserBookmarkBaseSerializer` has already declared `titlte` and `fancy_title` as attributes so there is technically no need for us to declare the attributes again in this serializer.

Reply:
This is no longer relevant after https://github.com/discourse/discourse/pull/16335#discussion_r844760034